notebooks/text_models/solutions/load_text.ipynb

```diff
@@ -0,0 +1,1935 @@
+{
```

Comment:
<p>Maybe explain why we set seed here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>If we can fix the seed at the beginning and not in the local function that would be better. Maybe by using something like the following at the beginning of the file?</p><p> </p><pre class="ql-syntax" spellcheck="false">SEED = 0
tf.keras.utils.set_random_seed(
    SEED
)
</pre>

Reply:
<p>Updated the instruction. <code>seed</code> is a required argument here for this function when specifying <code>validation_split</code> to ensure that there is no overlap between the training and validation subset . I updated the markdown and per Benoit's suggestion, also put it above.</p>